packages/shared/src/schemas.ts

```diff
@@ -171,6 +171,7 @@ interface BaseQuoteDetails {
   lowLiquidityWarning: boolean | undefined;
   estimatedDurationSeconds: number;
   estimatedPrice: string;
+  recommendedSlippageTolerancePercent: number; 
```

Comment:
maybe we should already add this to the changelog, even if its a static value for now? @acdibble

Reply:
Added to changelog